pyproject.toml

```diff
@@ -110,3 +111,9 @@ extend-exclude = '''
   src/codemodder/_version.py
 )/
 '''
+
+[coverage-threshold]
+line_coverage_min = 93.5
+    [coverage-threshold.modules."src/core_codemods/"]
```

Comment:
I think this can be at the same level of indentation as above.

Reply:
you're right! I copied the format [exactly](https://github.com/DeanWay/coverage-threshold?tab=readme-ov-file#config) as shown but they must have an indent mistake